tests/golden_config_infra/templates/sample_golden_config_db.j2

```diff
@@ -0,0 +1,16 @@
+{% set portchannels= [] %}
+{% for pc, value in PORTCHANNEL.items() %}
+    {% set _ = portchannels.append(pc) %}
+{% endfor %}
+
+{
+    "NEW_FEATURE": {
```

Comment:
New config_db table?

Reply:
It is just a test table which can be replaced with table of new features.